### PR TITLE
🔀🌐🇫🇷 first fr_FR translations

### PR DESCRIPTION
**Pull request by mattnounet**
Translation from EN to FR done up to line 75

### CODE REVIEW

**Comment by KokosnussDEV** [comment]
Is this still a WiP?

**Comment by mattnounet** [comment]
French translation finished.

File: src/i18n/fr_FR.ts [hunk 1]
```diff
@@ -935,204 +594,136 @@ export default {
...
-        5: 15,
+        0: 1,
+        2: 20,
+        6: 19,
     },
-    vehicleBuildings: [0, 3, 5, 6, 8, 9, 11, 12, 13, 14, 15, 16, 17, 18],
-    cellBuildings: [5],
+    vehicleBuildings: [0, 2, 5, 6, 13, 14, 18, 19, 20],
+    cellBuildings: [6, 16, 19],
     cellExtensions: [
```
Comment (JRH-1997) [inline src/i18n/fr_FR.ts:659]: You need to add the right extensions <BuildingType>_<ExtensionID>

File: src/i18n/fr_FR.ts [hunk 2]
```diff
@@ -1145,89 +736,86 @@ export default {
     },
     buildingIcons: [
         'fire',
-        'rss',
-        'hospital',
-        'clinic-medical',
         'graduation-cap',
-        'shield-alt',
+        'ambulance'
+        '',
```
Comment (JRH-1997) [inline src/i18n/fr_FR.ts:741]: You can find the right icons at [https://fontawesome.com/icons?d=gallery&m=free](https://fontawesome.com/icons?d=gallery&m=free)

File: src/i18n/fr_FR.ts [hunk 3]
```diff
@@ -0,0 +1,821 @@
...
+        {
+            caption: 'VIRT',
+            color: '#770000',
+            coins: 25,
+            credits: 19_200,
+            minPersonnel: 1,
+            maxPersonnel: 4,
+            wtank: 0,
+            special: 'Nécessaire dès lors que vous avez construit 7 bases de pompiers',
+            schooling: 'Véhicules risques chimiques',
```
Comment (Suchty112) [inline src/i18n/fr_FR.ts:163]: add this Centre de secours - Véhicules risques chimiques . what we need here is the training as it is in the course window, as with the alliance.

File: src/i18n/fr_FR.ts [hunk 4]
```diff
@@ -0,0 +1,821 @@
...
+        },
+        {
+            caption: 'VPC',
+            color: '#791515',
+            coins: 25,
+            credits: 25_500,
+            minPersonnel: 1,
+            maxPersonnel: 6,
+            wtank: 0,
+            schooling: 'Commandement mobile',
```
Comment (Suchty112) [inline src/i18n/fr_FR.ts:200]: Centre de secours - Commandement mobile

File: src/i18n/fr_FR.ts [hunk 5]
```diff
@@ -0,0 +1,821 @@
...
+        },
+        {
+            caption: 'VGRIMP',
+            color: '#ca3516',
+            coins: 25,
+            credits: 19_000,
+            minPersonnel: 2,
+            maxPersonnel: 5,
+            wtank: 0,
+            schooling: 'IMP (Intervention en Milieu Périlleux)',
```
Comment (Suchty112) [inline src/i18n/fr_FR.ts:229]: Centre de secours - IMP (Intervention en Milieu Périlleux)

File: src/i18n/fr_FR.ts [hunk 6]
```diff
@@ -1145,89 +736,86 @@ export default {
...
-        'stethoscope',
         'shield-alt',
-        'clinic-medical',
-        'plane',
+        '',
+        'fire',
         'shield-alt',
+        'ambulance',
     ],
     pois: [
```
Comment (JRH-1997) [inline src/i18n/fr_FR.ts:760]: Not the right name for in example 'Station-service' I will sent the whole list to you.

File: src/i18n/fr_FR.ts [hunk 7]
```diff
@@ -0,0 +1,821 @@
...
+        },
+        {
+            caption: 'VPL',
+            color: '#bb2222',
+            coins: 25,
+            credits: 10_000,
+            minPersonnel: 3,
+            maxPersonnel: 3,
+            wtank: 0,
+            schooling: 'PLG (Plongeur)',
```
Comment (Suchty112) [inline src/i18n/fr_FR.ts:249]: Centre de secours - PLG (Plongeur)

File: src/i18n/fr_FR.ts [hunk 8]
```diff
@@ -0,0 +1,821 @@
...
+        },
+        {
+            caption: 'Choucas',
+            color: '#ca1616',
+            coins: 30,
+            credits: 300_000,
+            minPersonnel: 2,
+            maxPersonnel: 4,
+            wtank: 0,
+            schooling: 'Licence de pilote',
```
Comment (Suchty112) [inline src/i18n/fr_FR.ts:271]: Poste de police - Licence de Pilote

File: src/i18n/fr_FR.ts [hunk 9]
```diff
@@ -0,0 +1,821 @@
...
+        },
+        {
+            caption: 'Equipe cynophile',
+            color: '#aa2222',
+            coins: 25,
+            credits: 7_000,
+            minPersonnel: 1,
+            maxPersonnel: 2,
+            wtank: 0,
+            schooling: 'Maître chien',
```
Comment (Suchty112) [inline src/i18n/fr_FR.ts:282]: Poste de police - Maître chien

File: src/i18n/fr_FR.ts [hunk 10]
```diff
@@ -0,0 +1,821 @@
...
+        },
+        {
+            caption: 'Unité motocycliste',
+            color: '#aa2255',
+            coins: 18,
+            credits: 2_500,
+            minPersonnel: 1,
+            maxPersonnel: 1,
+            wtank: 0,
+            schooling: 'Formation motocycliste',
```
Comment (Suchty112) [inline src/i18n/fr_FR.ts:294]: Poste de police - Formation Motocycliste

File: src/i18n/fr_FR.ts [hunk 11]
```diff
@@ -70,736 +70,442 @@ export default {
...
+        name: 'Paramètres généraux',
         labelInMenu: {
-            title: 'Label instead of icon in menu',
+            title: 'Titre au lieu d\'une icone dans le menu',
             description:
-                'Displays a simple label in the navigation bar instead of the LSSM logo',
+                'Un simple titre dans la barre de navigation au lieu du logo LSSM',
         },
     },
     vehicles: [
```
Comment (JRH-1997) [inline src/i18n/fr_FR.ts:86]: Vehicles need to be changed in 
vehicles: **{**
0: {
and so for every id:  
id: {
caption: '',
color: '',

**Review by Suchty112 — CHANGES_REQUESTED** [review]
so the first review. then also take over the one from jrh, then i look further

**Review by JRH-1997 — CHANGES_REQUESTED** [review]
Please look at my comments, I will sent a full list of the POI.

File: src/i18n/fr_FR.ts [hunk 12]
```diff
@@ -0,0 +1,821 @@
...
+                    duration: '7 jours',
+                },
+                {
+                    caption: 'Unité nautique',
+                    credits: 100_000,
+                    coins: 20,
+                    duration: '7 jours',
+                },
+            ],
+            levelcost: ['1. 10.000', '2. 50.000', '3.-39. 100.000'],
```
Comment (Suchty112) [inline src/i18n/fr_FR.ts:319]: 3.-16

File: src/i18n/fr_FR.ts [hunk 13]
```diff
@@ -0,0 +1,821 @@
...
+            maxBuildings: '2.700',
+            maxLevel: 39,
```
Comment (Suchty112) [inline src/i18n/fr_FR.ts:321]: 16

File: src/i18n/fr_FR.ts [hunk 14]
```diff
@@ -0,0 +1,821 @@
...
+            special: 'Le prix de vos postes augmente lorsque vous en possédez 25. Ainsi, le rythme de progression est constant une fois que vous possédez un grand flux de revenus à ces niveaux de jeu. La formule actuelle pour calculer le prix des postes est la suivante : <code>100.000+(200.000*LOG<sub>2</sub>(Number of existing fire stations − 22))</code>.',
+            startPersonnel: 10,
+            startVehicles: ['Fourgon Pompe-Tonne', 'Fourgon Pompe-Tonne Léger'],
```
Comment (Suchty112) [inline src/i18n/fr_FR.ts:324]: FPT, FPTL

File: src/i18n/fr_FR.ts [hunk 15]
```diff
@@ -0,0 +1,821 @@
...
+            credits: 100_000,
+            extensions: [
+                ...new Array(10).fill({
+                    caption: 'Cellule de prison',
+                    credits: 25_000,
+                    coins: 5,
+                    duration: '7 jours',
+                }),
+            ],
+            levelcost: ['1. 10.000', '2. 50.000', '3.-39. 100.000'],
```
Comment (Suchty112) [inline src/i18n/fr_FR.ts:457]: 3.-16

File: src/i18n/fr_FR.ts [hunk 16]
```diff
@@ -0,0 +1,821 @@
...
+            maxBuildings: '1200 postes de police/gendarmerie',
+            maxLevel: 14,
```
Comment (Suchty112) [inline src/i18n/fr_FR.ts:459]: 16

File: src/i18n/fr_FR.ts [hunk 17]
```diff
@@ -0,0 +1,821 @@
...
+                'Véhicule de patrouille': [8],
+                'Unité motocycliste': [20],
+                'Choucas': [18],
+                'Équipe cynophile': [19],
+            },
+            color: '#00ac00',
+        },
+    },
+    small_buildings: {
+        0: 1,
```
Comment (Suchty112) [inline src/i18n/fr_FR.ts:653]: 0: 18

**Review by Suchty112 — CHANGES_REQUESTED** [review]
and other review

File: src/i18n/fr_FR.ts (hunk 14 above)
Comment (Suchty112) [reply src/i18n/fr_FR.ts:324]: the same for the Centre de Première intervention

File: src/i18n/fr_FR.ts [hunk 18]
```diff
@@ -70,736 +70,442 @@ export default {
...
-            minPersonnel: 1,
-            maxPersonnel: 2,
-            schooling: 'Police - Sheriff',
-            shownSchooling: 'Sheriff',
+            schooling: 'Formation motocycliste',
+            shownSchooling: 'Policie motocycliste',
         },
+                
     ],
     buildings: [
```
Comment (JRH-1997) [inline src/i18n/fr_FR.ts:299]: Same as vehicles
buildings: **{**
id: {

And change the credits to a number without the _ so 100_00 will be 100000, the same at vehicles

File: src/i18n/fr_FR.ts [hunk 19]
```diff
@@ -769,26 +790,25 @@ export default {
...
         'Supermarché (petit)',
         'Supermarché (grand)',
-        'Station service',
+        'Station-service',
         'École',
         'Musée',
         'Centre commercial',
         'Garage automobile',
-        'Sortie d\'autoroute',
+        'Sortie d’autoroute',
```
Comment (JRH-1997) [inline src/i18n/fr_FR.ts:798]: It removed the escape, please add the escape again

File: src/i18n/fr_FR.ts (hunk 16 above)
Comment (mattnounet) [reply src/i18n/fr_FR.ts:458]: This building have only 14 level, not 16 like the others

File: src/i18n/fr_FR.ts [hunk 20]
```diff
@@ -769,26 +790,25 @@ export default {
...
-        '',
         'Théâtre',
-        'F$ete foraine',
+        'Fête foraine',
         'Rivière',
         'Petit aéroport (piste)',
-        'Grand aéroport (piste))',
-        'Terminal d\’aéroport',
+        'Grand aéroport (piste)',
+        'Terminal d’aéroport',
```
Comment (JRH-1997) [inline src/i18n/fr_FR.ts:811]: also escape

**Review by JRH-1997 — CHANGES_REQUESTED** [review]
Please escaspe that. Sorry, I forgot that in my list.

File: src/i18n/fr_FR.ts [hunk 21]
```diff
@@ -650,13 +649,35 @@ export default {
         },
     },
     small_buildings: {
-        0: 1,
+        0: 18,
         2: 20,
         6: 19,
     },
     vehicleBuildings: [0, 2, 5, 6, 13, 14, 18, 19, 20],
     cellBuildings: [6, 16, 19],
```
Comment (Suchty112) [inline src/i18n/fr_FR.ts:657]: remove 16 , we dont need this here

File: src/i18n/fr_FR.ts [hunk 22]
```diff
@@ -650,13 +649,35 @@ export default {
...
+        '6_1',
+        '6_2',
+        '6_3',
+        '6_4',
+        '6_5',
+        '6_6',
+        '6_7',
+        '6_8',
+        '6_9',
+        '16_0',
```
Comment (Suchty112) [inline src/i18n/fr_FR.ts:669]: here also remove the 16tens

**Review by Suchty112 — CHANGES_REQUESTED** [review]
Otherwise it looks good. The things are still changing and the one from JRH and we are done with it :P

**Review by Suchty112 — APPROVED** [review]

**Comment by KokosnussDEV** [comment]
Wait, there is one unresolved thingy

**Comment by Suchty112** [comment]
> Wait, there is one unresolved thingy: [#176 (comment)](https://github.com/LSS-Manager/LSSM-V.4/pull/176#discussion_r513442227)

he is right. fits already as he did it ^^

**Review by KokosnussDEV — APPROVED** [review]

**Review by KBOE2 — APPROVED** [review]

**Pull request merged**